CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [1.1.1] - 2019-10-25
+
+### Fixed
+
+- `distribution_policy_zones` ignoring [#37]
```

Comment:
```suggestion
- Fix bug with `distribution_policy_zones` forcing permadiff. [#37]
```

Reply:
Also, please don't put in the release numbers. You can just add this under "Unreleased"